cypress/integration/26-register_registration/creator_navigates_to_resource_type_tab/navigate_to_resource_type.js

```diff
@@ -0,0 +1,276 @@
+import { USER_WITH_AUTHOR } from '../../../support/constants';
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+import { JOURNAL_SUBTYPES, BOOK_FIELDS, RESOURCE_TYPES } from '../../../support/data_testid_constants';
+
+const doiLink = 'https://doi.org/10.1126/science.169.3946.635';
+const filename = 'example.txt';
+
+// Feature: Creator navigates to Resource Type tab
+
+// Common steps
+Given('Creator begins registering a Registration in the Wizard', () => {
+  cy.login(USER_WITH_AUTHOR);
+  let scenario = '';
+  if (window.testState.currentScenario.tags && window.testState.currentScenario.tags.length > 0) {
+    scenario = window.testState.currentScenario.tags[0].name;
+  } else {
+    scenario = window.testState.currentScenario.name;
+    if (scenario.includes('(example')) {
+      scenario = scenario.substring(0, scenario.indexOf('(example')).trim();
+    }
+  }
+
+  switch (scenario) {
+    case '@395':
+    case '@1625':
+    case '@1656':
+    case '@1659':
+    case '@1694':
+    case '@2021':
+    case 'Creator sees fields for Norwegian Science Index (NVI) incompatible Resource subtype':
+      cy.startRegistrationWithLink(doiLink);
+      cy.get('[data-testid=registration-link-next-button]').should('be.enabled');
+      cy.get('[data-testid=registration-link-next-button]').click({ force: true });
+      break;
+    default:
+      cy.startRegistrationWithFile(filename);
+      cy.get('[data-testid=registration-file-start-button]').should('be.enabled');
+      cy.get('[data-testid=registration-file-start-button]').click({ force: true });
+  }
+});
+When('they navigate to the Resource Type tab', () => {
+  cy.get('[data-testid=nav-tabpanel-resource-type').click({ force: true });
+});
+And('they click the Save button', () => {
+  cy.get('[data-testid=button-save-registration]').click({ force: true });
+  cy.get('[data-testid=button-save-registration]').should('be.enabled');
+  cy.get('[data-testid=button-next-tab]').click({ force: true });
+  cy.get('[data-testid=button-previous-tab]').click({ force: true });
+});
+And('they select the Resource type "Contribution to journal"', () => {
+  cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-context-type-Journal]').click({ force: true });
+});
+And('they select Resource type Book', () => {
+  cy.get('[data-testid=publication-context-type]').click({ force: true }).type(' ');
+  cy.get('[data-testid=publication-context-type-Book]').click({ force: true });
+});
+Then('they see a Search box for "Publisher name"', () => {
+  cy.get('[data-testid=publisher-search-field]').should('be.visible');
+});
+And('they see a checkbox for "Is this a textbook?"', () => {
+  cy.get('[data-testid=is-textbook-checkbox]').should('be.visible');
+});
+And('they select the Resource Type', (dataTable) => {

```

Comment:
hvilke resource types er det som testes her? det blir ikke duplikat av linjer 50-57 og 108-111?

Reply:
`Part of book/report`. Denne verdien må ligge i en tabell i scenarioet pga `/`

Reply:
Det er ikke en dupllikat, men scenarioene kunne vært skrevet med verdier i en tabell for å forenkle koden.

Reply:
så dataTable her er egentlig lik `Part of book/report`?

Reply:
Ja